### DISCUSSION TITLE
post/css-striped-list

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#23] [Announcements]
# post/css-striped-list

Mark Heath's Development Blog

https://markheath.net/post/css-striped-list

**Comment by m-weintraub** [comment]
Thanks, Mark.  I did two small extensions.  One was to use bootstrap color code for the background (`background-color: #f8f9fa;`).  I also used fontawesome ul (fa-ul) to have symbols for bullets, which required a small bit of padding on the fa-li to get the "bullets" to align.

`ul.striped-list > li > .fa-li {
  padding-top: 6px;
}`
